public/js/app/components/RoundApplicationInfo.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Button } from 'reactstrap'
+import i18n from '../../../../i18n'
+import { INFORM_IF_IMPORTANT_INFO_IS_MISSING } from '../util/constants'
+
+const LABEL_MISSING_INFO = { en: INFORM_IF_IMPORTANT_INFO_IS_MISSING[0], sv: INFORM_IF_IMPORTANT_INFO_IS_MISSING[1] }
+
+const RoundApplicationInfo = ({
+  roundHeader,
+  selectedRoundHeader,
+  userLanguage,
+  round,
+  showRoundData,
+  courseHasRound,
+  fundingType,
+}) => {
+  const { courseRoundInformation: translate, courseLabels: labels } = i18n.messages[userLanguage]
+
+  const openApplicationLink = ev => {
```

Comment:
This function has not been removed from `public/js/app/components/RoundInformationOneCol.jsx`, but I think it can, right?